src/main/java/com/ghostchu/peerbanhelper/module/impl/webapi/PBHClientStatus.java

```diff
@@ -36,6 +36,7 @@ public NanoHTTPD.Response handle(NanoHTTPD.IHTTPSession session) {
         for (Downloader downloader : getServer().getDownloaders()) {
             Map<String, Object> map = new HashMap<>(2);
             map.put("name", downloader.getName());
+            map.put("type", downloader.getDownloaderName());
```

Comment:
Use `downaloaderName` instead for getting unique id, type is reserved for downloader type (qBittorrent, Transmission, Deluge etc.)

Reply:
and the `getDownloaderName` is a bad name, it just return the downloader type...

Reply:
> and the `getDownloaderName` is a bad name, it just return the downloader type...

no, it returns Downloader Name: 

![image](https://github.com/PBH-BTN/PeerBanHelper/assets/30802565/f3bcad07-822c-4e63-a33e-f7bd6afa4636)